3396.xml

```diff
@@ -63,7 +63,7 @@ ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEAL
 				<Mandatory>Mandatory</Mandatory>
 				<Type>String</Type>
 				<RangeEnumeration>1-64 bytes</RangeEnumeration>
-				<Units>byte</Units>
+				<Units>B</Units>
```

Comment:
I think this resource should actually be unitless. The enumeration should be “1..64”.

Reply:
I agree with Ari. The enumeration should say "1..64" and the <Units> element should be empty.

Reply:
@hannestschofenig , @akeranen 
 The change to add "1..64" is done but the group still needs to decide if we need to remove the Units.
This Object was submitted by OpenAIS, do we need to check with them this suggestion about unit-less?

Reply:
The [RangeEnumeration field](http://www.openmobilealliance.org/release/LightweightM2M/V1_1_1-20190617-A/HTML-Version/OMA-TS-LightweightM2M_Core-V1_1_1-20190617-A.html#12-1-0-D1-Object-Template) should not contain word ”Bytes”, just the numbers and delimiters.

Reply:
@akeranen we need to evaluate all the Objects from OpenAIS in case other resources have the same problem.